feat: create database models

penguinuux opened the pull request:
Initiated flask migrate;
Created address, cities, messages, states, users models;
Created users_messages and users_risks tables.

penguinuux committed:
feat: create database models

Initiated flask migrate;
Created address, cities, messages, states, users models;
Created users_messages and users_risks tables.

JOSUE-ASV reviewed (approved):
it's all right

SidnySilva commented on app/models/message_model.py:
```diff
@@ -0,0 +1,22 @@
+
+@dataclass
+class MessageModel(db.Model):
+    id: int
```
id shouldn't be returned for user

penguinuux replied:
Done!

SidnySilva commented on app/models/user_model.py:
```diff
@@ -0,0 +1,29 @@
+
+@dataclass
+class UserModel(db.Model):
+    id: int
```
id shouldn't be returned.

penguinuux replied:
Done!

marcelosj3 commented on app/models/address_model.py:
```diff
@@ -0,0 +1,16 @@
+
+@dataclass
+class AddressModel(db.Model):
+    id: int
```
Do we need to show id when requesting the address? I don't think this is necessary since the we will probably use the name to make this relationship.

penguinuux replied:
Done!

marcelosj3 commented on app/models/city_model.py:
```diff
@@ -0,0 +1,19 @@
+
+@dataclass
+class CityModel(db.Model):
+    id: int
```
Same as above

penguinuux replied:
Done!

marcelosj3 reviewed (changes requested):
Along with the changes i've requested, to maintain a structural pattern in our code, we can use, strictly in this order, `isort .` and then `black .` in any terminal so it can both sort the imports and organize the code. Might as well be useful to import a `.vscode` folder with some configs, but we can check about this later.

marilia-calado reviewed (commented):
The changes pointed by @marcelosj3 and @soudyy should be taken in consideration before the merge.

luizpcardoso reviewed (changes requested):
sidny and marcelo pointed out some details that I agree with

pablo-adriano committed:
fix: remove id from some models

Removed id initiation from MessageModel, UserModel, AdressModel and
CityModel;
Sorted all file imports with 'isort' and formated with 'black'.

SidnySilva reviewed (changes requested):
All right

luizpcardoso reviewed (approved):
ok!
